### PR TITLE
Sort JSON Data

### PR DESCRIPTION
**Pull request by hmajid2301**
#### Short description of what this resolves:

Sort the JSON (settings) files before they are uploaded.

#### Changes proposed in this pull request:

- If the sort setting is enabled
- Sort every JSON file before upload

**Fixes**: #998 

#### How Has This Been Tested?

As per my slack messages I have been unable to test it locally.
Perhaps someone can test it on their machine (locally). 

#### Checklist:

- [x] I have read the [contribution](https://github.com/shanalikhan/code-settings-sync/blob/master/CONTRIBUTING.md#setup-extension-locally) guidelines.
- [x] My change requires a change to the documentation and GitHub Wiki.
- [ ] I have updated the documentation and Wiki accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -134,6 +134,11 @@
           "default": true,
           "description": "%ext.config.removeExtensions%"
         },
+        "sync.sortAlphabetically": {
```
Comment (shanalikhan) [inline package.json]: can you put it as global settings instead of env. settings.

**Review by shanalikhan — CHANGES_REQUESTED** [review]
Hello @hmajid2301 !

Sorry i was away from last month. I'm going to review this PR. Pls complete the changes so i can merge into next release branch

File: package.json (hunk 1 above)
Comment (hmajid2301) [reply package.json]: How can i do that ?

File: README.md [hunk 2]
```diff
@@ -154,6 +154,7 @@ You can customize the settings in gist settings like:
     "sync.forceDownload": false,
     "sync.forceUpload": false,
     "sync.quietSync": false,
+    "sync.sortAlphabetically", false,
```
Comment (nandalopes) [inline README.md]: Change comma to colon

"sync.sortAlphabetically": false,

File: package.json (hunk 1 above)
Comment (hmajid2301) [reply package.json]: What about this comment ?

**Comment by shanalikhan** [comment]
Can you fix this or i can close this PR.
Thanks for the work, the only thing remaining is to put the configuration in Global Settings of settings sync

Here : https://github.com/shanalikhan/code-settings-sync#global-settings

Instead of the current place of configuration you have added.

Pls fix and let me know so i can add in upcoming release

**Comment by hmajid2301** [comment]
Ok I will fix the issues

File: package.json (hunk 1 above)
Comment (shanalikhan) [reply package.json]: `sortAlphabetically` needs to be moved into 

https://github.com/shanalikhan/code-settings-sync#global-settings

In this file
https://github.com/shanalikhan/code-settings-sync/blob/master/src/models/customConfig.model.ts

**Pull request merged**